api/src/controllers/courseController.js

```diff
@@ -9,30 +9,34 @@ const { changeFormat } = require('../helpers/filehelpers')
 // @route   GET /api/courses
 // @access  Public
 const getCourses = async (req, res) => {
-  const pageSize = 10
-  const page = Number(req.query.pageNumber) || 0
+  const pageSize = 6
+  const page = Number(req.query.pageNumber) || 1
   const order = req.query.order || 'ASC'
 
-  const courses = await db.Courses.findAll({
-    offset: page,
+  const courses = await db.Courses.findAndCountAll({
```

Comment:
seems like this is not filtering category try to use it similar to library one

Reply:
filtering category is implemented at server side similar to library.